internal/server/handlers_test.go

```diff
@@ -1458,3 +1458,47 @@ func TestAPI_LoginResponse(t *testing.T) {
 	assert.Equal(t, loginResp.Name, "steve")
 	assert.Equal(t, loginResp.PasswordUpdateRequired, false)
 }
+
+func TestListGrants(t *testing.T) {
```

Comment:
Is this different from the test cases in `grants_test.go` ?

Reply:
This is testing the `ListGrants` handler, right? Can we move this into `grants_test.go` next to the other tests for this function?

It would also be good to include the particular case in the name of the test. From reading the test, it seems like it's testing inherited grants, so I don't think I understand why this wouldn't be another sub test in `TestAPI_ListGrants_InheritedGrants`